src/Illuminate/Foundation/Configuration/ApplicationBuilder.php

```diff
@@ -191,8 +191,7 @@ public function withMiddleware(callable $callback)
     {
         $this->app->afterResolving(HttpKernel::class, function ($kernel) use ($callback) {
             $middleware = (new Middleware)
-                ->auth(redirectTo: fn () => route('login'))
-                ->guest(redirectTo: fn () => route('dashboard'));
+                ->auth(redirectTo: fn () => route('login'));
```

Comment:
Sometimes, the `login` named route of the application does not exist, or it is named something else. Maybe this PR can be expanded to redirect to `/` if that route does not exist?

Reply:
The other place where this is hard-coded is in the default `AuthenticationException` handler (`Illuminate\Foundation\Exceptions\Handler`)